go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-azurerm
 
 require (
-	github.com/Azure/azure-sdk-for-go v40.3.0+incompatible
+	github.com/Azure/azure-sdk-for-go v40.4.0+incompatible
```

Comment:
from what I can tell this is in v40.3.0 of the Azure SDK too - could we use that version for now? (we can update the SDK in another PR if it's needed)